pkg/yopass/yopass_test.go

```diff
@@ -147,6 +148,21 @@ func TestEncrypt(t *testing.T) {
 	}
 }
 
+type invalidFile struct{}
+
+func (invalidFile) Read(p []byte) (n int, err error) { return 0, fmt.Errorf("Broken I/O") }
+
+func TestEncryptWithInvalidFile(t *testing.T) {
+	_, err := yopass.Encrypt(invalidFile{}, "somekey")
+	if err == nil {
+		t.Fatal("expected error, got none")
+	}
+	want := "could not copy data: Broken I/O"
+	if err.Error() != "could not copy data: Broken I/O" {
```

Comment:
I guess you could use the `want` variable here?

Reply:
Yes thanks! 👍